### PR TITLE
Fix misspellings in documentation.

### PR DESCRIPTION
**Pull request by ka1bi4**
## Proposed change

Fixed some misspellings in documentation.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/ads.markdown [hunk 1]
```diff
@@ -18,6 +18,8 @@ The ADS (automation device specification) describes a device-independent and fie
 
 There is currently support for the following device types within Home Assistant:
 
+- [Configuration](#configuration)
+- [Service](#service)
```
Comment (frenck) [inline source/_integrations/ads.markdown:22]: This is not a Table of contents. Please remove these 2 added items.

File: source/_integrations/alexa.smart_home.markdown [hunk 2]
```diff
@@ -40,9 +40,58 @@ Steps to Integrate an Amazon Alexa Smart Home Skill with Home Assistant:
...
+  - [Script](#script)
+  - [Sensor](#sensor)
+  - [Switch](#switch)
+  - [Timer](#timer)
+  - [Vacuum](#vacuum)
 - [Alexa Web-Based App](#alexa-web-based-app)
 - [Troubleshooting](#troubleshooting)
+  - [Binary Sensor not available in Routine Trigger](#binary-sensor-not-available-in-routine-trigger)
+  - [Token Invalid and no Refresh Token Available](#token-invalid-and-no-refresh-token-available)
 - [Debugging](#debugging)
```
Comment (frenck) [inline source/_integrations/alexa.smart_home.markdown]: Please revert this TOC change

File: source/_integrations/alexa.smart_home.markdown [hunk 3]
```diff
@@ -426,41 +475,68 @@ Home Assistant supports the following integrations through Alexa using a Smart H
...
+  - [Script](#script)
+  - [Sensor](#sensor)
+  - [Switch](#switch)
+  - [Timer](#timer)
+  - [Vacuum](#vacuum)
+- [Alexa Web-Based App](#alexa-web-based-app)
+- [Troubleshooting](#troubleshooting)
+  - [Binary Sensor not available in Routine Trigger](#binary-sensor-not-available-in-routine-trigger)
+  - [Token Invalid and no Refresh Token Available](#token-invalid-and-no-refresh-token-available)
+- [Debugging](#debugging)
```
Comment (frenck) [inline source/_integrations/alexa.smart_home.markdown:539]: This is not a TOC

**Review by frenck — CHANGES_REQUESTED** [review]
Looks good! Left a couple of small comments, @ka1bi4.

Could you take a look? Thanks 👍

File: source/_integrations/ads.markdown (hunk 1 above)
Comment (ka1bi4) [reply source/_integrations/ads.markdown:22]: Ok, remove it in next commit.

File: source/_integrations/alexa.smart_home.markdown (hunk 3 above)
Comment (ka1bi4) [reply source/_integrations/alexa.smart_home.markdown:539]: Fix it

File: source/_integrations/alexa.smart_home.markdown (hunk 2 above)
Comment (ka1bi4) [reply source/_integrations/alexa.smart_home.markdown]: Fix it

**Comment by ka1bi4** [comment]
Hi, @frenck. 
Thanks for comments. I tried to fix all, hope it's ok.

File: source/_integrations/ads.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/ads.markdown:22]: This comment isn't processed. Unresolving it.

File: source/_integrations/alexa.smart_home.markdown (hunk 2 above)
Comment (frenck) [reply source/_integrations/alexa.smart_home.markdown]: This has not been resolved. the whole change is till there, please revert this.

File: source/_integrations/alexa.smart_home.markdown (hunk 3 above)
Comment (frenck) [reply source/_integrations/alexa.smart_home.markdown:539]: The TOC is still there?

**Comment by frenck** [comment]
Almost 2 months and no response on the last comment. Going to close this PR as stale for now. Feel free to re-open when ready to work on it again 👍

**Pull request closed without merging**